src/components/calcite-tip-manager/calcite-tip-manager.tsx

```diff
@@ -94,6 +94,9 @@ export class CalciteTipManager {
 
   connectedCallback() {
     this.setUpTips();
+    if (this.el.hasAttribute("tabindex")) {
+      this.tabindex = this.el.getAttribute("tabindex");
```

Comment:
I think we should be consistent with the `calcite-action` and not set a tabIndex on the host.

Reply:
This may also be related: https://github.com/ionic-team/stencil/issues/1008

Reply:
and this. https://github.com/ionic-team/stencil/issues/1623